bin/phpcs-excludelist.json

```diff
@@ -0,0 +1,604 @@
+[
+    "bin\/build-asset-cdn-json.php",

```

Comment:
Let's get rid of the extraneous backslashes, and use tabs for indenting. I pushed a464f02e60d944c726a46b078111d4e8ec4f5f13 for that if you want to pull that into this PR.

Reply:
Cool, will do. I just piped the textlist into PHP and let it json_encode with pretty print do magic. Hold no option on the formatting.